packages/cardano-services/test/jest-setup/rebuild-test-db.sh

```diff
@@ -18,6 +18,7 @@ yarn --cwd "$WORKSPACE_ROOT" build
 yarn --cwd "$PACKAGES_DIR"/e2e wait-for-network
 yarn --cwd "$PACKAGES_DIR"/e2e test:wallet
 yarn --cwd "$PACKAGES_DIR"/e2e test:long-running delegation-rewards.test.ts
+TL_LEVEL="${TL_LEVEL:=info}" node "$SCRIPT_DIR/mint-handles.js"
```

Comment:
Probably organizing the `mint-handles.js` as an e2e test could help simplifying a bit.

Since the actual target of the test would be not to assert some feature behavior but just to _set up_ data for fixtures, we could add a new set of tests in e2e package which do not run in CI (something like `yarn test:fixtures`) to run this script, more any other we may need in the future.

Reply:
Clear, thank you.
Just in case, better to leave as is in this PR and open a new task to add the _after hook_ you described plus the required refactoring.